src/main/java/org/nibor/autolink/LinkExtractor.java

```diff
@@ -27,11 +27,13 @@ public static Builder builder() {
 
     /**
      * Extract the links from the input text. Can be called multiple times with different inputs (thread-safe).
-     *
-     * @param input the input text, must not be {@code null}
+     * @param input - the input text
```

Comment:
Don't add the `-`. Javadoc already adds this. Adding `-` in the source results in doubled dashes like `- -`.